app.js

```diff
@@ -0,0 +1,73 @@
+
+'use strict';
+
+var port = process.env.PORT || 3000;
```

Comment:
Because the example app currently defaults to `5000`, could we change this default to `5000` as well?